OrangeHRMTests/PageObjects/Pages/DashboardPage.cs

```diff
@@ -0,0 +1,42 @@
+﻿using OpenQA.Selenium;
+using OrangeHRMTests.Common.WebElements;
+
+namespace OrangeHRMTests.PageObjects.Pages
+{
+    public class DashboardPage : BasePage
+    {
+        private MyWebElement ArrowButton = new MyWebElement(By.XPath("//i[@class='oxd-icon bi-caret-down-fill oxd-userdropdown-icon']"));
```

Comment:
please review locators for all elements in your framework. As I can see, they can be improved
example, ArrowButton can be written like //*[contains(@class, 'dropdown-tab')]//i

all these 'oxd-icon bi-caret-down-fill' - are not super stable, highly likely it's class attributes set for specific CSS properties by dev. if they change it, for example to ' 'oxd-icon bi-caret-down-fill**-filled**' - you will have your locators failed

Reply:
Fixed